scala/scala.bzl

```diff
@@ -579,12 +573,13 @@ def _scala_junit_test_impl(ctx):
         main_class = "org.junit.runner.JUnitCore",
         jvm_flags = launcherJvmFlags + ctx.attr.jvm_flags,
         args = test_suite.suite_class,
-        run_before_binary = "",
-        run_after_binary = "",
     )
 
     return _scala_binary_common(ctx, cjars, rjars)
 
+_launcher_template = {
+  "_java_stub_template": attr.label(default=Label("@java_stub_template//file")),
```

Comment:
This is more of a question to you/Oscar. Looking at commit: https://github.com/bazelbuild/rules_scala/commit/0e4ff2d4e4d1d84f7c42a188ae75ded45cdea545

I don't have a good sense of when to use //external versus @... since other parts of file use @bazel_tools (although I presume because those aren't actually setup in the WORKSPACE and assumed to be premapped).

Also, test/BUILD still references @scala//:scala-library and I wasn't sure if that was an oversight or intentional demonstration that rules_scala 'clients' should still be using @scala.